pycds/alembic/versions/22819129a609_convert_collapsed_vars_mv_to_native_.py

```diff
@@ -44,41 +45,51 @@ def create_dependent_objects():
 
 
 def upgrade():
-    #  We could do this with a DROP ... CASCADE, but that invites disasters.
-    drop_dependent_objects()
+    engine = op.get_bind().engine
+    if not matview_exists(
```

Comment:
Would it make sense to have some kind of error message if the matview exists?

Reply:
It's an expected possible condition, so not an error. But it's not a bad idea to log it as an info-level message.